cfssl/models/policy_use.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License MIT (https://opensource.org/licenses/MIT).
+
+
+class PolicyUse(object):
+    """ It provides a Certificate Use policy compatible with CFSSL """
+
+    def __init__(self, name, code):
```

Comment:
I'm not sure if this is something you want to do, but you could case a lot of code in this class and others by requiring these options to be sent in as keyword args.
```
class PolicyUse(object):

    def __init__(self, *args, **kwargs):
        self.__dict__.update(**kwargs)

a = PolicyUse(name='some_name', code='some_code')

print a.code
```

Output is:
some_code